### ISSUE TITLE
password issue

### ISSUE THREAD

**Issue by hectorprats** [jvranish/docker-https-ssh-tunnel#2]
Hi, when I try to up with the sh file, it asks for a password.

Which is the password??

**Comment by hectorprats** [comment]
"tunnel" does not work

**Comment by Redsandro** [comment]
I have the same problem. There is no password. It doesn't accept our ssh key. @jvranish any idea what might cause this?

```
debug1: Offering public key: RSA SHA256:xxx /home/redsandro/.ssh/id_rsa.pub
debug3: send_pubkey_test
debug3: send packet: type 50
debug2: we sent a publickey packet, wait for reply
debug3: receive packet: type 51
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug2: we did not send a packet, disable method
debug3: authmethod_lookup keyboard-interactive
debug3: remaining preferred: password
debug3: authmethod_is_enabled keyboard-interactive
debug1: Next authentication method: keyboard-interactive
debug2: userauth_kbdint
debug3: send packet: type 50
debug2: we sent a keyboard-interactive packet, wait for reply
debug3: receive packet: type 51
debug1: Authentications that can continue: publickey,password,keyboard-interactive
debug3: userauth_kbdint: disable: no info_req_seen
debug2: we did not send a packet, disable method
debug3: authmethod_lookup password
debug3: remaining preferred: 
debug3: authmethod_is_enabled password
debug1: Next authentication method: password
```

**Comment by blaketwist** [comment]
Same here. Also after I created a new key, with surely no password. Any solution on this @jvranish ?

**Comment by jvranish** [comment]
I believe this is fixed by #1
